component/remote/http/http.go

```diff
@@ -34,10 +38,11 @@ func init() {
 
 // Arguments control the remote.http component.
 type Arguments struct {
-	URL           string        `river:"url,attr"`
-	PollFrequency time.Duration `river:"poll_frequency,attr,optional"`
-	PollTimeout   time.Duration `river:"poll_timeout,attr,optional"`
-	IsSecret      bool          `river:"is_secret,attr,optional"`
+	URL                  string        `river:"url,attr"`
+	PollFrequency        time.Duration `river:"poll_frequency,attr,optional"`
+	PollTimeout          time.Duration `river:"poll_timeout,attr,optional"`
+	IsSecret             bool          `river:"is_secret,attr,optional"`
+	FallbackCacheEnabled bool          `river:"fallback_cache_enabled,attr,optional"`
```

Comment:
Would calling this `cache` instead of `fallback_cache` lose meaning? (I'm concerned about the length of the name being annoying to type)

Reply:
Yeah, I mulled over this one a bit. My reasoning was that it might be confused with a cache that optimizes queries?

Reply:
Perhaps just a `fallback` block?
```river
fallback { enabled = true }
```

Reply:
Maybe @clayton-cornell would have opinions for what name be commonly understood here?

Reply:
I missed this one.

Do we need `Enabled` in the bool name? Id' probably opt for either `Fallback` or `FallbackCache`. It's a bool, so if it's set to t`true` it should be clear it's enabled.... or it seems that way to me.